test/com/facebook/buck/features/apple/project/testdata/project_generated_scheme_with_env_variables_and_expand_setting/Apps/TestApp.xcworkspace/xcshareddata/xcschemes/TestApp.xcscheme.expected

```diff
@@ -0,0 +1 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?><Scheme LastUpgradeVersion="9999" version="1.7"><BuildAction buildImplicitDependencies="YES" parallelizeBuildables="YES"><BuildActionEntries><BuildActionEntry buildForAnalyzing="YES" buildForArchiving="YES" buildForProfiling="YES" buildForRunning="YES" buildForTesting="YES"><BuildableReference BlueprintIdentifier="E66DC04E0E6F340F00000000" BlueprintName="TestApp" BuildableIdentifier="primary" BuildableName="TestApp.app" ReferencedContainer="container:Apps.xcodeproj"/></BuildActionEntry><BuildActionEntry buildForAnalyzing="YES" buildForArchiving="NO" buildForProfiling="NO" buildForRunning="NO" buildForTesting="YES"><BuildableReference BlueprintIdentifier="E66DC04E79595B5200000000" BlueprintName="TestAppTests" BuildableIdentifier="primary" BuildableName="TestAppTests.xctest" ReferencedContainer="container:Apps.xcodeproj"/></BuildActionEntry></BuildActionEntries></BuildAction><TestAction buildConfiguration="Debug" shouldUseLaunchSchemeArgsEnv="NO"><Testables><TestableReference skipped="NO"><BuildableReference BlueprintIdentifier="E66DC04E79595B5200000000" BlueprintName="TestAppTests" BuildableIdentifier="primary" BuildableName="TestAppTests.xctest" ReferencedContainer="container:Apps.xcodeproj"/></TestableReference></Testables><MacroExpansion><BuildableReference BlueprintIdentifier="E66DC04E0E6F340F00000000" BlueprintName="TestApp" BuildableIdentifier="primary" BuildableName="TestApp.app" ReferencedContainer="container:Apps.xcodeproj"/></MacroExpansion><EnvironmentVariables><EnvironmentVariable isEnabled="YES" key="TEST_ENV" value="SET"/></EnvironmentVariables></TestAction><LaunchAction buildConfiguration="Debug" launchStyle="0"><BuildableProductRunnable><BuildableReference BlueprintIdentifier="E66DC04E0E6F340F00000000" BlueprintName="TestApp" BuildableIdentifier="primary" BuildableName="TestApp.app" ReferencedContainer="container:Apps.xcodeproj"/></BuildableProductRunnable></LaunchAction><ProfileAction buildConfiguration="Release" shouldUseLaunchSchemeArgsEnv="YES"><BuildableProductRunnable><BuildableReference BlueprintIdentifier="E66DC04E0E6F340F00000000" BlueprintName="TestApp" BuildableIdentifier="primary" BuildableName="TestApp.app" ReferencedContainer="container:Apps.xcodeproj"/></BuildableProductRunnable></ProfileAction><AnalyzeAction buildConfiguration="Debug"/><ArchiveAction buildConfiguration="Release" revealArchiveInOrganizer="YES"/></Scheme>
```

Comment:
Is this file used in your tests?

Reply:
Yes, this is a plist (XML with Apple flavour) that describe schema. My PR affect only schema files.